src/node.h

```diff
@@ -484,6 +484,8 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
  */
 NODE_EXTERN void AtExit(void (*cb)(void* arg), void* arg = 0);
 
+extern bool config_preserve_symlinks;
```

Comment:
There should be no need to expose `config_preserve_symlinks` in node.h, should there?  Also, you don't seem to be defining it in any .cc file.